system/xmllib/libxml_xpathprocessor.cpp

```diff
@@ -1064,6 +1499,36 @@ class CEsdlScriptContext : public CInterfaceOf<IEsdlScriptContext>
     {
         return getSectionNode(name, nullptr);
     }
+
+    xmlNodePtr ensureCopiedSection(const char *target, const char *source, bool replace)
+    {
+        sanityCheckSectionName(target);
+        sanityCheckSectionName(source);
+        if (isEmptyString(source)||isEmptyString(target))
+            return nullptr;
```

Comment:
These checks may be backwards. If strpbrk, used by sanityCheckSectionName, is anything like other standard library functions that behave badly when passed null, null should be checked first (or as part of the sanity check if it makes sense).